website/templates/profile/account.mako

```diff
@@ -129,9 +129,15 @@
                 <div id="deactivateAccount" class="panel panel-default">
                     <div class="panel-heading clearfix"><h3 class="panel-title">Deactivate Account</h3></div>
                     <div class="panel-body">
-                        <p class="alert alert-warning"><strong>Warning:</strong> This action is irreversible.</p>
+                        %if not requestedDeactivation:
+                            <p class="alert alert-warning"><strong>Warning:</strong> This action is irreversible.</p>
+                        %endif
                         <p>Deactivating your account will remove you from all public projects to which you are a contributor. Your account will no longer be associated with OSF projects, and your work on the OSF will be inaccessible.</p>
-                        <a class="btn btn-danger" data-bind="click: submit, css: success() === true ? 'disabled' : ''">Request deactivation</a>
+                        %if not requestedDeactivation:
+                             <a class="btn btn-danger" data-bind="click: submit, css: success() === true ? 'disabled' : ''">Request deactivation</a>
```

Comment:
The minor downside to this method is that the view is different on reloading the page than when the request successfully returns. Better would be to ensure that knockout has access to requestedDeactivation and handle the view from there. While you're waiting to hear back from the server, deactivating the button is proper, but once you do, it would be better to switch to the requestedDeactivation branch.

Also, while we're here, what happens if you get an error from the server?

If this will take more than a day or so, let me know, because having this change is better than waiting too long on the ideal version.